src/cfnlint/data/ExtendedSpecs/all/04_property_values/aws_config.json

```
@@ -41,6 +41,13 @@
       "ValueType": "AWS::Config::ConfigRule.SourceDetail.MessageType"
     }
   },
+  {
+    "op": "add",
+    "path": "/PropertyTypes/AWS::Config::ConfigRule/Properties/Description/Value",
```

[actually just figured this out myself last month](https://github.com/aws-cloudformation/cfn-python-lint/pull/1654/files)

```suggestion
    "path": "/ResourceTypes/AWS::Config::ConfigRule/Properties/Description/Value",
```